src/magic_folder/web.py

```diff
@@ -515,8 +513,13 @@ def _is_authorized(request, get_auth_token):
     if len(authorization) > 1:
         return False
     auth_token = get_auth_token()
-    expected = u"Bearer {}".format(auth_token).encode("ascii")
+    expected = b"Bearer {}".format(auth_token)
```

Comment:
might want to leave the former version since `bytes` on python3 doesn't have a `.format()` .. no action required since we don't actually support py3 though :/